Formula/artillery.rb

```diff
@@ -7,6 +7,11 @@ class Artillery < Formula
   sha256 "dd2270e6056923595b6dfa59c1b8548d5bc92539ce984b9fa2d0584cd2fdf6cc"
   license "MPL-2.0"
 
+  livecheck do
+    url :stable
+    regex(/href=.*?artillery[._-]v?(\d+(?:\.\d+)+[._-]\d+)\.t/i)
```

Comment:
```suggestion
    regex(%r{["'][^"' ]*?/artillery[._-]v?(\d+(?:[.-]\d+)+)\.t}i)
```

The tarball URLs are actually found in some JavaScript code on the page, so `href=.*?` isn't quite doing what you would expect here. This suggested regex should match the tarball URLs without escaping the bounds of the related string. I'll have to revisit the `Npm` strategy in the not-too-distant future.